Description: Fix paths in logroate config to match obey FHS
Author: Gernot Schulz <gernot@intevation.de>
Last-Update: 2016-07-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/contrib/logrotate/intelmq
+++ b/contrib/logrotate/intelmq
@@ -10,7 +10,7 @@
     notifempty
     sharedscripts
     postrotate
-        sudo -u intelmq /usr/local/bin/intelmqctl reload --quiet
+        sudo -u intelmq /usr/bin/intelmqctl reload --quiet
     endscript
 }
 
@@ -21,6 +21,6 @@
         notifempty
     sharedscripts
     postrotate
-        sudo -u intelmq /usr/local/bin/intelmqctl reload --quiet
+        sudo -u intelmq /usr/bin/intelmqctl reload --quiet
     endscript
 }
